__tests__/app.test.js

```diff
@@ -10,14 +10,42 @@ afterAll(() => {
   db.end();
 });
 
-describe.only("GET /api/topics", () => {
+describe("GET /api", () => {
+  test("should return an object containing a message key", () => {
+    return request(app)
+      .get(`/api`)
+      .expect(200)
+      .then((response) => {
+        expect(response.body).toMatchObject({ message: "all ok" });
```

Comment:
Make sure that if you're sending custom http messages you use something like "msg" to distinguish your errors from PSQL errors which use "message"